### PR TITLE
DEVX-2044: Document ccloud-stack option to reuse existing environment

### PR DESCRIPTION
**Pull request by ybyzek**

### CODE REVIEW

File: ccloud/docs/ccloud-stack.rst [hunk 1]
```diff
@@ -119,6 +107,31 @@ Destroy a ccloud-stack
...
+
+   Then create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION``, as shown below.
+
+   .. code:: bash
+
+      CLUSTER_CLOUD=aws CLUSTER_REGION=us-west-2 ./ccloud_stack_create.sh
+
+#. By default, a new `ccloud-stack`` creates a new environment. To reuse an existing environment, create the ``ccloud-stack`` and override the parameter ``ENVIRONMENT`` with an existing environment ID, as shown below.
+
+   .. code:: bash
```
Comment (tikimims) [inline ccloud/docs/ccloud-stack.rst]:
```suggestion
   .. code-block:: bash
```

File: ccloud/docs/ccloud-stack.rst [hunk 2]
```diff
@@ -119,6 +107,31 @@ Destroy a ccloud-stack
...
+
+#. By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. To create resources in another cloud provider or region other than the default, use the |ccloud| CLI to view the available cloud providers and regions:
+
+   .. code:: bash
+
+      ccloud kafka region list
+
+   Then create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION``, as shown below.
+
+   .. code:: bash
```
Comment (tikimims) [inline ccloud/docs/ccloud-stack.rst]:
```suggestion
   .. code-block:: bash
```

File: ccloud/docs/ccloud-stack.rst [hunk 3]
```diff
@@ -119,6 +107,31 @@ Destroy a ccloud-stack
...
 
+.. _ccloud-stack-options:
+
+================
+Advanced Options
+================
+
+#. By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. To create resources in another cloud provider or region other than the default, use the |ccloud| CLI to view the available cloud providers and regions:
+
+   .. code:: bash
```
Comment (tikimims) [inline ccloud/docs/ccloud-stack.rst]:
```suggestion
   .. code-block:: bash
```

File: ccloud/docs/ccloud-stack.rst [hunk 4]
```diff
@@ -119,6 +107,31 @@ Destroy a ccloud-stack
...
 
 
+.. _ccloud-stack-options:
+
+================
+Advanced Options
+================
+
+#. By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. To create resources in another cloud provider or region other than the default, use the |ccloud| CLI to view the available cloud providers and regions:
+
```
Comment (tikimims) [inline ccloud/docs/ccloud-stack.rst]:
```suggestion
By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. To create resources in another cloud provider or region other than the default, complete the following steps:

#.  View the available cloud providers and regions using the |ccloud| CLI:

```

File: ccloud/docs/ccloud-stack.rst [hunk 5]
```diff
@@ -119,6 +107,31 @@ Destroy a ccloud-stack
...
+   .. code:: bash
+
+      ccloud kafka region list
+
+   Then create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION``, as shown below.
```
Comment (tikimims) [inline ccloud/docs/ccloud-stack.rst]:
```suggestion
#. Create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION`` by running the following command:
```

File: ccloud/docs/ccloud-stack.rst [hunk 6]
```diff
@@ -119,6 +107,31 @@ Destroy a ccloud-stack
...
+
+   .. code:: bash
+
+      CLUSTER_CLOUD=aws CLUSTER_REGION=us-west-2 ./ccloud_stack_create.sh
+
+#. By default, a new `ccloud-stack`` creates a new environment. To reuse an existing environment, create the ``ccloud-stack`` and override the parameter ``ENVIRONMENT`` with an existing environment ID, as shown below.
+
```
Comment (tikimims) [inline ccloud/docs/ccloud-stack.rst]:
```suggestion
By default, a new ``ccloud-stack`` creates a new environment. To reuse an existing environment, create the ``ccloud-stack`` and override the parameter ``ENVIRONMENT`` with an existing environment ID, as shown in the following example:```

File: ccloud/docs/ccloud-stack.rst [hunk 7]
```diff
@@ -68,25 +68,13 @@ Setup
...
-      ccloud kafka region list
-
-   Then create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION``, as shown below.
-
-   .. code:: bash
-
-      CLUSTER_CLOUD=aws CLUSTER_REGION=us-west-2 ./ccloud_stack_create.sh
-
-#. In addition to creating all the resources in |ccloud| with associated service account and ACLs, it also generates a local configuration file with all above connection information, useful for other demos/automation. View this file at ``stack-configs/java-service-account-<SERVICE_ACCOUNT_ID>.config``. It resembles:
+#. In addition to creating all the resources in |ccloud| with associated service account and ACLs, running ``ccloud-stack`` also generates a local configuration file with all above connection information, useful for other demos/automation. View this file at ``stack-configs/java-service-account-<SERVICE_ACCOUNT_ID>.config``. It resembles:
```
Comment (tikimims) [inline ccloud/docs/ccloud-stack.rst]: @ybyzek For "with all above connection information," how about "with all the previous connection information" or "with all the earlier connection information?"

File: ccloud/docs/ccloud-stack.rst [hunk 8]
```diff
@@ -68,25 +68,13 @@ Setup
 Create a ccloud-stack
 ---------------------
 
-#. By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. If this is the target provider and region, then create the stack by calling the bash script :devx-examples:`ccloud_stack_create.sh|ccloud/ccloud-stack/ccloud_stack_create.sh`.
+#. By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. If this is the target provider and region, then create the stack by calling the bash script :devx-examples:`ccloud_stack_create.sh|ccloud/ccloud-stack/ccloud_stack_create.sh`. For additional options when configuring your ``ccloud-stack``, see :ref:`ccloud-stack-options`.
```
Comment (tikimims) [inline ccloud/docs/ccloud-stack.rst]:
```suggestion
#. By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. If this is the target provider and region, create the stack by calling the bash script :devx-examples:`ccloud_stack_create.sh|ccloud/ccloud-stack/ccloud_stack_create.sh`. For more options when configuring your ``ccloud-stack``, see :ref:`ccloud-stack-options`.
```

**Review by tikimims — COMMENTED** [review]
Okay @ybyzek. I made a few suggestions/comments.

File: ccloud/docs/ccloud-stack.rst (hunk 7 above)
Comment (ybyzek) [reply ccloud/docs/ccloud-stack.rst]: Refined a bit

File: ccloud/docs/ccloud-stack.rst [hunk 9]
```diff
@@ -119,6 +107,39 @@ Destroy a ccloud-stack
...
+
+By default, the ``cloud-stack`` utility creates resources in the cloud provider ``aws`` in region ``us-west-2``. To create resources in another cloud provider or region other than the default, complete the following steps:
+
+#. View the available cloud providers and regions using the |ccloud| CLI:
+
+   .. code-block:: bash
+
+      ccloud kafka region list
+
+#. Create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION``, as shown below.
```
Comment (tikimims) [inline ccloud/docs/ccloud-stack.rst]:
```suggestion
#. Create the ``ccloud-stack`` and override the parameters ``CLUSTER_CLOUD`` and ``CLUSTER_REGION``, as shown in the following example:
```

**Review by tikimims — APPROVED** [review]
Approved, but with one comment/suggestion

**Pull request merged**